blockchain/types/tx_internal_data.go

```diff
@@ -219,7 +219,7 @@ func (t TxType) IsChainDataAnchoring() bool {
 
 // IsRPCExcluded returns true if the submission of such tx type is excluded via RPC.
 func (t TxType) IsRPCExcluded() bool {
```

Comment:
Is there any reason for preserving this method?
As far as I know, it was only for the ChainDataAnchoring Tx type.

Reply:
Good. Let's remove.